skiplang/skjson/ts/src/skjson.ts

```diff
@@ -107,9 +107,16 @@ function interpretPointer<T extends Internal.CJSON>(
     case Type.Array: {
       const aPtr = hdl.access.SKIP_SKJSON_asArray(ptr);
       const length = hdl.access.SKIP_SKJSON_arraySize(aPtr);
-      return Array.from({ length }, (_, idx) =>
+      const array = Array.from({ length }, (_, idx) =>
         interpretPointer(hdl, hdl.access.SKIP_SKJSON_at(aPtr, idx)),
       );
+      return Object.freeze(
+        Object.defineProperty(array, sk_frozen, {
+          enumerable: false,
+          writable: false,
+          value: true,
+        }),
+      ) as Exportable;
```

Comment:
There is redundancy between this code and `sk_freeze`, which would be good to de-dup. @skiplabsdaniel do you know if this duplication is necessary due to package dependencies?